src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.db;
+
+import core.basesyntax.models.Bet;
+import core.basesyntax.models.User;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Bet> BETS = new ArrayList<>();
+    public static final List<User> USERS = new ArrayList<>();
```

Comment:
though the modifiers are the same - these lists are not conceptually constants, there names should be lowercase